lib/front-end-builds-notifier.js

```diff
@@ -30,53 +30,51 @@ module.exports = CoreObject.extend({
     return { sha, branch };
   },
 
-  notify() {
+  async notify() {
     const html = this.getIndexContent();
     const { branch, sha } = this.gitInfo();
     const signature = this.sign(html);
     const app_name = this.readConfig('app');
     const data = { app_name, branch, sha, signature, html };
     const plugin = this;
```

Comment:
I'm _assuming_ this scoping work around is no longer required?

Reply:
Also probably fine to just leave it if you're going for a low touch approach

Reply:
Yeah true. It makes me feel all nostalgic though, so gonna leave it in 😂